src/block-management/block-holder.js

```diff
@@ -163,6 +165,28 @@ export class BlockHolder extends React.Component<PropsType, StateType> {
 		);
 	}
 
+	getAccessibilityLabel() {
+		const { title, attributes, name } = this.props;
+
+		if ( name === getUnregisteredTypeHandlerName() ) { // is the block unrecognized?
+			return title; //already localized
+		}
+
+		const blockName = sprintf(
+			/* translators: accessibility text. %s: block name. */
+			_x( '%1$s block' ),
```

Comment:
Should this be `__` instead of `_x`?
And since there's just one variable, `%d` should be enough?